devtools_m1/cypress.json

```diff
@@ -1,5 +1,7 @@
 {
   "projectId": "35f86d",
   "ignoreTestFiles": "*.js",
-  "baseUrl": "http://main.magento.localhost:3005"
+  "baseUrl": "http://main.magento.localhost:3005",
+  "defaultCommandTimeout": 15000,
+  "requestTimeout": 15000
```

Comment:
Excessive values, but we're really not looking to measure the performance of Magento. Occasionally a cy.contains(...) or xhr takes just a little longer than the timeouts.